packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -195,6 +198,33 @@ describe('CheckoutModule', () => {
             });
         });
 
+        describe('getBasket ::', () => {
+            let config;
+
+            beforeEach(() => {
+                config = {
+                    method: 'get',
+                    headers: {
+                        'Content-Type': 'application/json',
+                        'Accept-Tenant': payload.tenant,
+                        'Accept-Language': payload.language
+                    },
+                    timeout: payload.timeout
+                };
+
+                axios.get = jest.fn(() => Promise.resolve({ data: basketDelivery }));
+            });
```

Comment:
Same as before. We could remove this `beforeEach` in this case. Sorry, didn't catch it in my original review.

Reply:
Fixed